lib/ae_mdw/db/name.ex

```diff
@@ -30,13 +30,19 @@ defmodule AeMdw.Db.Name do
     |> map_ok!(&Validate.id!/1)
   end
 
-  def owned_by(owner_pk) do
+  def owned_by(owner_pk, _active? = true) do
     %{
-      actives: collect_vals(Model.ActiveNameOwner, owner_pk),
+      names: collect_vals(Model.ActiveNameOwner, owner_pk),
       top_bids: collect_vals(Model.AuctionOwner, owner_pk)
     }
   end
 
+  def owned_by(owner_pk, _active? = false) do
+    %{
+      names: collect_vals(Model.InactiveNameOwner, owner_pk)
```

Comment:
I don't think we should ever return unlimited lists of results in a single page, this is easily exploitable. But we were already doing it for active names so I'll leave it up to you. Perhaps we could think of a new endpoint?